locales/en-US/app.l20n

```diff
@@ -53,8 +53,8 @@
 <experimentListPageHeader "Ready for Takeoff!">
 <experimentListPageSubHeader  "Pick the features you want to try. <br> Check back soon for more experiments.">
 
-<experimentListInactiveHover "Get Started">
-<experimentListActiveHover "Manage">
+<experimentListInactiveHover "Just Launched">
+<experimentListActiveHover "Enabled">
```

Comment:
I think we need to change the name of any localized string if its value changes. Asking about this now